autogen/safer-cluster/variables.tf.tmpl

```diff
@@ -344,18 +341,6 @@ variable "resource_usage_export_dataset_id" {
   default     = ""
 }
 
-variable "enable_network_egress_export" {
```

Comment:
Could we plumb these to the underlying module?

Reply:
done